### ISSUE TITLE
[P2][Sourcing][Accessibility][ATL testing][Table]: Search label is not read in JAWS only

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10613]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Search label is not read in JAWS only as part of table, it used to read before , but in latest version its not reading 

#### Can you handle that on the application side
no

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

ng: 15
platform: 0.43.x

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
turn on JAWS -> focus on search field -> it just reads as "type in text"

#### Please provide relevant source code (if applicable).
replicable on JAWS only on : https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic

**Comment by nikolay-kolarov** [comment]
Hello,

There is an aria-label="Search" on the search field, but it is overwritten by an aria-labelledby attribute that points to the title.
I see two options:
- Remove the aria-labelledby pointing to the title. The table element needs to be labelled with the title. For example the structure in UI5: https://ui5.sap.com/#/entity/sap.m.Table/sample/sap.m.sample.TableScrollToIndex/code
- If the aria-labelledby pointing to the title is required and agreed for previous tickets, then the text "search" can be added in a hidden node and referenced as well in aria-labelledby (e.g aria-labelledby="idElement1 idElement2").

Best Regards,
Nikolay

**Comment by droshev** [comment]
We should be careful with the first option as we may break some agreement from earlier issues.

**Comment by g-cheishvili** [comment]
It would be better in my opinion to have aria-label if provided(adding input to the table toolbar) and if not, use aria-labelledby pointing to title
